Examples/Run/Reconstruction/Common/RecTruthTracks.cpp

```diff
@@ -33,9 +33,8 @@
 using namespace Acts::UnitLiterals;
 using namespace ActsExamples;
 
-int main(int argc, char* argv[]) {
-  GenericDetector detector;
-
+int recTruthTracks(int argc, char* argv[],
```

Comment:
```suggestion
int runRecTruthTracks(int argc, char* argv[],
```
Suggestion: as above.

Reply:
Changed.